src/DurableFunctions.TypedInterfaces/Example/DurableFunctions.TypedInterfaces.Example.csproj

```diff
@@ -9,8 +9,8 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" Version="2.2.0" />
-    <PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage" Version="1.8.5" />
-    <PackageReference Include="Microsoft.Azure.DurableTask.Core" Version="2.5.2" />
+    <PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage" Version="2.0.0-preview.2" />
```

Comment:
Shouldn't this be preview.3?

Reply:
It has now been updated after the package publish